email/email_test.go

```diff
@@ -0,0 +1,40 @@
+package email
+
+import (
+	"os"
+	"testing"
+)
+
+type config struct {
+	SMTPFrom          string `long:"smtpfrom" description:"From address to use on outbound mail"`
+	SMTPHost          string `long:"smtphost" description:"SMTP hostname/ip and port, e.g. mail.example.com:25"`
+	SMTPUsername      string `long:"smtpusername" description:"SMTP username for authentication if required"`
+	SMTPPassword      string `long:"smtppassword" description:"SMTP password for authentication if required"`
+	UseSMTPS          bool   `long:"usesmtps" description:"Connect to the SMTP server using smtps."`
+	SMTPSkipTLSVerify bool   `long:"smtpskiptlsverify" description:"Skip TLS verification when it establish connection to the SMTP server."`
+}
+
+func TestSMTPSendMail(t *testing.T) {
+	cfg := config{
+		SMTPHost:     os.Getenv("SMTPHost"),
+		SMTPUsername: os.Getenv("SMTPUsername"),
+		SMTPPassword: os.Getenv("SMTPPassword"),
+		SMTPFrom:     os.Getenv("SMTPFrom"),
```

Comment:
None of these environment variables are set by default for me.  Should I be setting them before the test?

Reply:
this test will be skipped, if don't to provide real credentials.
to run it 
```
SMTPHost="smtp.fastmail.com:587" SMTPUsername="**"  SMTPPassword="**" SMTPFrom="**" go test github.com/decred/dcrstakepool/email -run TestSMTPSendMail
```

Reply:
I see. Maybe a comment explaining that is warranted.